app/routes/project/posts/post.js

```diff
@@ -32,6 +32,16 @@ export default Ember.Route.extend({
   },
 
   actions: {
+    save(post) {
+      post.save().catch((error) => {
```

Comment:
@WenInCode, good call on adding `return` to the save function. One problem though, I deleted all references to the route's save function from the post-title test, and it still passes. This is because the component's save function sets the post.title field, and the last assert is only checking the value of post.title. The test doesn't check that the post's title has been persisted to the database. Do you have any idea how to pull that off? Maybe at least test that the route action is being called somehow? Not sure how to do that.